src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -65,7 +65,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.1.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.20.1.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
```

Comment:
Kindly update the version in the [netcore csproj](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.Netcore.csproj) as well

Reply:
Done. Thank you.